src/stashmanager.hpp

```diff
@@ -0,0 +1,16 @@
+#pragama once
+#include "HashTables.hpp"
```

Comment:
Esto faltaría actualizarse al nuevo nombre del archivo, que es `hashtable.hpp`. También falta el include de `stash.hpp`.

```suggestion
#include "hashtable.hpp"
#include "stash.hpp"
```